lib/responsys/api/client.rb

```diff
@@ -30,6 +40,19 @@ def api_method(action, message = nil, response_type = :hash)
         end
       end
 
+      def run(exception_raising = false)
+        old_raise_exceptions = @raise_exceptions
+        @raise_exceptions = exception_raising
+        begin
+          yield(self)
+        rescue DisabledException => e
+          raise e if exception_raising
```

Comment:
since you're assigning `exception_raising` to `@raise_exceptions` above, why not use `@raise_exceptions` in the conditional here?